src/components/admin/services/timer/TimerPage.jsx

```diff
@@ -0,0 +1,192 @@
+import { useState } from "react";
+import Timer from "./Timer";
+import { v4 as uuidv4 } from "uuid";
+
+export default function TimerPage() {
+  const [timerArray, setTimerArray] = useState([]);
+  const [boxVisbiility, setBoxVisibility] = useState(false);
+  const [days, setDays] = useState(0);
+  const [hours, setHours] = useState(0);
+  const [minutes, setMinutes] = useState(0);
+  const [seconds, setSeconds] = useState(0);
+  const [timerName, setTimerName] = useState("New Timer");
+  const [isInputValid, setIsInputValid] = useState(false);
+
+  const addTimer = () => {
+    if (hours > 23 || minutes > 59 || seconds > 59) {
+      setIsInputValid(true);
+      return;
+    }
+    const time = new Date();
+    time.setDate(time.getDate() + parseInt(days));
+    time.setHours(time.getHours() + parseInt(hours));
+    time.setMinutes(time.getMinutes() + parseInt(minutes));
+    time.setSeconds(time.getSeconds() + parseInt(seconds));
+
+    document.getElementById("timerName").value = "";
```

Comment:
this should not be needed, there are surely better ways to handle values using state